examples/example_annubes.ipynb

```diff
@@ -40,18 +77,37 @@
    "source": [
     "import warnings\n",
     "\n",
-    "from stable_baselines3 import A2C  # ACER, PPO2\n",
+    "from stable_baselines3 import A2C\n",
     "from stable_baselines3.common.vec_env import DummyVecEnv\n",
     "\n",
     "warnings.filterwarnings(\"default\")\n",
     "\n",
-    "# Optional: PPO2 requires a vectorized environment to run\n",
-    "# the env is now wrapped automatically when passing it to the constructor\n",
-    "env = DummyVecEnv([lambda: AnnubesEnv()])\n",
-    "\n",
-    "model = A2C(\"MlpPolicy\", env, verbose=0)\n",
+    "# train agent\n",
+    "env = AnnubesEnv()\n",
+    "env_vec = DummyVecEnv([lambda: env])\n",
+    "model = A2C(\"MlpPolicy\", env_vec, verbose=0)\n",
     "model.learn(total_timesteps=20000, log_interval=1000)\n",
-    "env.close()"
+    "env_vec.close()\n",
+    "\n",
+    "# plot example trials with trained agent\n",
+    "data = ngym.utils.plot_env(env, num_trials=5, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
```

Comment:
it's 10 for all other plots in this notebook. I think for the sake of not confusing a user why the plot would look different, it would be good to stick to the same number in each case.


```suggestion
    "data = ngym.utils.plot_env(env, num_trials=10, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
```